packages/fxa-content-server/server/lib/sentry.js

```diff
@@ -5,12 +5,17 @@
 const _ = require('lodash');
 
 const config = require('./configuration');
-const SENTRY_SERVER_ERRORS_DSN = config.get('sentry.server_errors_dsn');
 const STACKTRACE_FRAME_LENGTH = 10;
 const RELEASE = require('../../package.json').version;
-const { tagCriticalEvent } = require('fxa-shared/tags/sentry');
+const {
+  tagCriticalEvent,
+  buildSentryConfig,
+  tagFxaName,
+} = require('fxa-shared/sentry');
+const logger = require('./logging/log')('sentry');
 
 const Sentry = require('@sentry/node');
+require('@sentry/tracing');
```

Comment:
Does this just work™? Looks like we have a mix of `require('@sentry/tracing');`, `import * as Tracing from '@sentry/tracing';`, and `  const Tracing = require('@sentry/tracing');` but maybe that's all intentional based on package needs.